src/Sulu/Bundle/ContactBundle/Resources/config/lists/contacts.xml

```diff
@@ -296,12 +296,13 @@
         </property>
 
         <group-concat-property
-            name="tagIds"
-            visibility="never"
+            name="tagNames"
+            visibility="yes"
             translation="sulu_tag.tags"
             glue=","
+            distinct="true"
```

Comment:
is this really required? I think distinct should not be needed here, but not sure 🤔